meson.build

```diff
@@ -0,0 +1,3 @@
+project('docker-mac-bridge')
+
+install_data('docker-mac-bridge', install_dir: 'bin', install_mode: 'rwxr-xr-x')
```

Comment:
I've never used a `meson.build` before. I'd prefer a `Makefile` honestly, to reduce dependencies.

Reply:
That makes sense. Meson is a fantastic (and fast!) build system especially for C/C++ code. I've used it for some internal projects, and was mainly leveraging its rich understand of system prefixes across platforms, which is a bit like using a sledgehammer to hit nail. Using Makefiles we'd have to manually manage the install prefixes; but since it is a single file... that's hardly a massive undertaking.